### ISSUE TITLE
Swap sort and filter

### ISSUE THREAD

**Issue by nikku** [helpers/handlebars-helper-compose#18]
From what I see in the source code, the sort step is applied after the actual filtering takes place. This disallows me to limit the number of results **after** sorting.

Is there any particular reasoning behind this?

I would expect filtering to be applied after sorting to give the user higher flexibility in what he wants to do.

**Comment by jonschlinkert** [comment]
makes sense. marking as a feature request as this is something that wasn't needed before.
